cvat/apps/quality_control/quality_reports.py

```diff
@@ -79,8 +81,72 @@ def from_dict(cls, d: dict):
         raise NotImplementedError("Must be implemented in the subclass")
 
 
-@define(kw_only=True)
-class AnnotationId(_Serializable):
+class _ReportNode(_Serializable):
+    _CHECK_SETATTR_FIELD_NAME = "_check_setattr"
+
+    _CACHED_FIELDS: Optional[List[str]] = None
```

Comment:
I don't quite understand; it seems that the code depends on this attribute being initialized in each derived class, but you removed all `_CACHED_FIELDS` assignments?

Reply:
The list of cached fields can be defined explicitly in each sublclass, but it's being derived automatically from the class declaration now in all cases in the `_find_cached_fields()` classmethod.

Reply:
I see. But in that case, why even have the `_CACHED_FIELDS` attribute?